### ISSUE TITLE
mix appsignal.diagnose: Appsignal.Mixfile.project/0 is undefined

### ISSUE THREAD

**Issue by jeffkreeftmeijer** [appsignal/appsignal-elixir#142]
```
** (UndefinedFunctionError) function Appsignal.Mixfile.project/0 is undefined (module Appsignal.Mixfile is not available)
```

https://app.intercom.io/a/apps/yzor8gyw/inbox/conversation/8348966448%3Fnsrc=slack

**Comment by tombruijn** [comment]
This appears to be happening the second time you run `mix appsignal.diagnose` if you only run this mix task. I would assume it works the first time, because it's still compiling then and the `Appsignal.Mixfile` module is available then. The second time it isn't because AppSignal is already compiled.

**Comment by tombruijn** [comment]
Fixed in #143
